src/classes/Recipe.js

```diff
@@ -26,26 +26,59 @@ class Recipe {
 
   getIngredientNames() {
     this.setIngredients();
-    return this.ingredients.map(ingredient => {
+    let ingredientNames = this.ingredients.map(ingredient => {
       return ingredient.name;
     });
+    // let formatted = ingredientNames.map((word) => {
+    //   return word[0].toUpperCase() + word.substring(1);
+    // });
+
+    // [0].toUpperCase() + ingredient.name.substring(1)
+    return ingredientNames
+    // return ingredientNames.join(', ')
   }
 
+  getIngredientTerms() {
+    let currentIngredients = this.getIngredientNames();
+    let ingredientsBroken = [];
+    let spaces = new RegExp(" ")
```

Comment:
This new RegExp is a good find. Good to learn how .test below works with it to find a space